packages/e2e-tests/src/specs/dashboard/settings/adminUser/customFonts.js

```diff
@@ -103,10 +100,14 @@ describe('Custom Fonts', () => {
     await addCustomFont(OPEN_SANS_CONDENSED_LIGHT_ITALIC_URL);
     await addCustomFont(OPEN_SANS_CONDENSED_BOLD_URL);
     await addCustomFont(OPEN_SANS_CONDENSED_LIGHT_URL);
-    const fonts = await getFontList();
-    await page.click('div[role=listbox] [role=option]:last-child');
-    const selected = await getSelectedFont();
-    expect(selected.name).toStrictEqual(fonts[2].name);
+
+    const font = await page.$('div[role=listbox] [role=option]:last-child');
+    await font.click();
```

Comment:
Something is happening here where the fonts don't get cleared and we're trying to re-add 

Causing this issue.

![should-select-font-when-clicked-2022-06-28T11-27-41](https://user-images.githubusercontent.com/62242/176173415-1ac58b94-3dc2-4470-8adf-34537de0d76d.jpg)

Reply:
The `should select font when clicked` test for checking the `aria-selected` value seems better suited for Jest unit tests tbh

Reply:
Removed --- can follow-up with a Jest Test